models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php

```diff
@@ -493,4 +493,18 @@ public function filterMultipleAssignments($data, $container, $params)
 
         return $data;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function getInputType() {
```

Comment:
does it return null when nothing is set? or does it return a empty array?

Reply:
Hard to decide in the base class. So be on the safe side.